boards/nucleo_f429zi/src/main.rs

```diff
@@ -57,6 +57,11 @@ struct NucleoF429ZI {
         VirtualMuxAlarm<'static, stm32f4xx::tim2::Tim2<'static>>,
     >,
     gpio: &'static capsules::gpio::GPIO<'static>,
+    /* uncomment to use an HD47880 LCD
+    lcd: &'static capsules::hd44780::HD44780<
+        'static,
+        VirtualMuxAlarm<'static, stm32f4xx::tim2::Tim2<'static>>,
+    >,*/
```

Comment:
I am also inclined not to merge these commented out capsule instantiations into master, as they clutter main.rs and most people are unlikely to use this particular capsule

Reply:
I removed them from the main.rs file.